src/prefabs/pageWithCreateDataTable.js

```diff
@@ -0,0 +1,12156 @@
+      'HAS_AND_BELONGS_TO_MANY',
+      'HAS_MANY',
+      'MULTI_FILE',
+      'AUTO_INCREMENT',
+      'COUNT',
+      'MULTI_IMAGE',
+      'PDF',
+      'RICH_TEXT',
+      'SIGNED_PDF',
+      'SUM',
+      'BOOLEAN_EXPRESSION',
+      'DATE_EXPRESSION',
+      'DATE_TIME_EXPRESSION',
+      'DECIMAL_EXPRESSION',
+      'INTEGER_EXPRESSION',
+      'MINUTES_EXPRESSION',
+      'PRICE_EXPRESSION',
+      'STRING_EXPRESSION',
+      'TEXT_EXPRESSION',
+      'MINUTES',
+      'ZIPCODE',
+    ];
+
+    return (
+      <>
+        <Header title="Configure data table" onClose={close} />
+        <Content>
+          <Field
+            label="Model"
+            error={
+              modelValidation && (
+                <Text color="#e82600">Selecting a model is required</Text>
+              )
+            }
+          >
+            <ModelSelector
+              onChange={value => {
+                setModelValidation(false);
+                setModelId(value);
+              }}
+              value={modelId}
+            />
+          </Field>
+          <Field
+            label="Columns in the data table"
+            error={
+              propertiesValidation && (
+                <Text color="#e82600">Selecting a property is required</Text>
+              )
+            }
+          >
+            <PropertiesSelector
+              modelId={modelId}
+              value={properties}
+              disabledKinds={disabledKinds}
+              onChange={value => {
+                setProperties(value);
+              }}
+            />
+          </Field>
+          <Field label="Create Form">
+            <CheckBox
+              label="Use the same properties as the data table in the create form"
+              checked={createFormUseDataProperties}
+              onChange={() => {
+                setCreateFormUseDataProperties(!createFormUseDataProperties);
+              }}
+            />
+            {!createFormUseDataProperties && (
+              <Field
+                label="Input fields in the create form"
+                error={
+                  createPropertiesValidation && (
+                    <Text color="#e82600">
+                      Selecting a property is required
+                    </Text>
+                  )
+                }
+              >
+                <PropertiesSelector
+                  modelId={modelId}
+                  value={createFormProperties}
+                  disabledNames={['created_at', 'updated_at', 'id']}
+                  disabledKinds={disabledKinds}
+                  onChange={value => {
+                    setCreatePropertiesValidation(!value.length);
+                    setCreateFormProperties(value);
+                  }}
+                />
+              </Field>
+            )}
+          </Field>
+        </Content>
+        <Footer
+          onSave={() => {
+            if (
+              !modelId ||
+              (createFormProperties.length < 1 &&
```

Comment:
Why is length < 1 being done here. Feels like a weird way to check if array is empty.

The above `!createFormProperties.length` seems like a better way to check it.

Same for the cases below this.

Reply:
That's a good feedback point. I'll try to keep using the shorter method from now on.